quirks/change-password-URLs.json

```diff
@@ -2,6 +2,7 @@
     "500px.com": "https://web.500px.com/settings/account/security",
     "amctheatres.com": "https://www.amctheatres.com/amcstubs/account",
     "autodesk.com": "https://accounts.autodesk.com/Profile/Security",
+    "deutsche-bank.de": "https://www.deutsche-bank.de/pfb/content/marktinformationen/passwort_vergessen.html",
```

Comment:
Is this a forgot password link or a change password link?

Reply:
It's a forgot password link, where you can request a new password.